addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -353,6 +353,9 @@ export class OdooEditor extends EventTarget {
         this.addDomListener(this.document, 'mousedown', this._onDoumentMousedown);
         this.addDomListener(this.document, 'mouseup', this._onDoumentMouseup);
 
+        // when you click outisde the parent iframe
+        this.addDomListener(this.document.defaultView, 'parentBlur', this._onParentWindowBlur);
```

Comment:
Wasn't it possible to do this with a standard event? Maybe something like 'click' on `window.top` but only if `window.top !== this.document.defaultView`?

Reply:
That's actually way better, Thanks for the suggestion !